.github/workflows/ci-pyright-fails.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: pip install -e ".[extras]"
```

Comment:
I think you forgot a line above this, right? That's why it did not find the pytest dependency.